.github/CODEOWNERS

```diff
@@ -1,2 +1 @@
-# This should match the team set up in https://github.com/orgs/opensearch-project/teams and include any additional contributors
-*   @opensearch-project/clients @Xtansia
\ No newline at end of file
+*   @dblock @Xtansia @VachaShah @aditjind
```

Comment:
Looks like @aditjind doesnt have write access which is why Codeowners is complaining. 
Can we add him as an external maintainer?

Reply:
Yes, he needs his maintain permission added back (would have previously had it via a team). I only have maintain perms myself, so I can't add him, needs an admin to do it.

Reply:
This has now been done 👍